react/Modal/Readme.md

```diff
@@ -66,6 +66,21 @@ const spacings = [
 </div>
 ```
 
+### mobileFullscreen
+
+If you have a modal filled with content and want it to occupy all the available space on mobile screen.
```

Comment:
If you want the modal to fill all the available space, without margin, on mobile screen.